src/screens/Settings/HSwitchProfile/HSwitchProfileSettings.res

```diff
@@ -1,56 +1,45 @@
 module MerchantDetailsSection = {
   @react.component
   let make = () => {
-    let fetchDetails = APIUtils.useGetMethod()
-    let (merchantInfo, setMerchantInfo) = React.useState(_ => Dict.make())
+    open HSwitchProfileSettingsEntity
     let (screenState, setScreenState) = React.useState(_ => PageLoaderWrapper.Success)
```

Comment:
Why default is Success?
Can you change this to Loading and on line 15 we can remove that.